src/plugins/stake-plugins/generic/policyAdapters/TarotPoolAdaptor.ts

```diff
@@ -512,14 +515,24 @@ export const makeTarotPoolAdapter = (policyConfig: StakePolicyConfig<TarotPoolAd
         }
       }
 
-      const allocations: PositionAllocation[] = []
-      const token0AccruedBalance = await tarot.getAccruedBalance(token0BorrowableContract, walletAddress)
-      const token0NativeAmount = token0AccruedBalance.toString()
-      allocations.push(getAllocation(token0, token0NativeAmount))
+      const leverageFloat = await tarot.getLeverage()
+      const leverageInt = Math.round(leverageFloat)
```

Comment:
Ditch the `leverageFloat` and just have `const leverage = Math.round(await tarot.getLeverage())`. Less code to read and simplifies the name. What do you think?

Reply:
For debugging, I prefer to see exactly what the tarot utils blackbox produces before touching it. Not a hill I'm will to die on and can be convinced to change it

Reply:
Meh, alright.